src/python/bot/tasks/task_creation.py

```diff
@@ -173,6 +176,10 @@ def create_variant_tasks_if_needed(testcase):
     if testcase.project_name != project_name:
       continue
 
+    # Don't create cross-platform variants for test cases with gestures.
+    if testcase.gestures and job.platform != testcase_platform:
```

Comment:
This means we dont try the testcase at all. i would prefer if we did try the testcase on other platforms, but without the gestures set. can shift this change to just like https://github.com/google/clusterfuzz/blob/master/src/python/bot/tasks/variant_task.py#L51

where we compare platform
if testcase.platform != environment.platform().lower():
  testcase.gestures = None (or [])

Reply:
Good call, this is a much better idea. I don't like the idea of modifying the gesture string since that could lead to some subtle breakages if we ever decided to put the test case in variant task, but addressed it in a similar way.